### PR TITLE
DEP Update dependency versions

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,9 +6,9 @@
     "license": "BSD-3-Clause",
     "require": {
         "silverstripe/recipe-plugin": "^1",
-        "silverstripe/recipe-cms": "4.x-dev",
-        "silverstripe/segment-field": "2.x-dev",
-        "silverstripe/userforms": "5.x-dev"
+        "silverstripe/recipe-cms": "4.9.x-dev",
+        "silverstripe/segment-field": "2.4.x-dev",
+        "silverstripe/userforms": "5.9.x-dev"
```
Comment (maxime-rainville) [inline composer.json:11]:
```suggestion
        "silverstripe/userforms": "5.10.x-dev"
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
We are shipping [Userform 5.10 in this recipe](https://github.com/silverstripe/recipe-form-building/blob/1.9.0-beta1/composer.json)

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json]: Updated

**Comment by emteknetnz** [comment]
Failing unit test is because of SwiftNailer and has been fixed separately.  Should pass once everything is merged

**Comment by maxime-rainville** [comment]
Merging despite failing build so we can the 4.9 branches tested against the correct branches.

**Pull request merged**